primerj/src/main/java/org/primer/primerj/PrimerjSettings.java

```diff
@@ -140,8 +140,10 @@ public int value() {
     public static final boolean ensureMinRequiredFee = true;
 
     public enum TransactionFeeMode {
-        Normal(1000000), High(2000000), Higher(5000000), TenX(10000000), TwentyX(20000000);
 
+        // Normal(1000000), High(2000000), Higher(5000000), TenX(10000000), TwentyX(20000000);
+        // Reduce number of modes and adjust fee by a factor of 100
```

Comment:
This line is best for the commit message instead of here. Here can say something like:
// Transaction fee rate per KB